apps/omg_watcher/test/omg_watcher/integration/block_getter_test.exs

```diff
@@ -204,24 +203,24 @@ defmodule OMG.Watcher.Integration.BlockGetterTest do
       "utxo_pos" => utxo_pos
     } = TestHelper.get_exit_data(exit_blknum, 0, 0)
 
-    {:ok, %{"status" => "0x1", "blockNumber" => eth_height}} =
-      Eth.RootChainHelper.start_exit(
-        utxo_pos,
-        txbytes,
-        proof,
-        alice.addr
-      )
-      |> Eth.DevHelpers.transact_sync!()
-
-    # Here we're waiting for passing of margin of slow validator(m_sv)
-    exit_processor_sla_margin = Application.fetch_env!(:omg_watcher, :exit_processor_sla_margin)
-    Eth.DevHelpers.wait_for_root_chain_block(eth_height + exit_processor_sla_margin, @timeout)
-
-    # Here we're manually submitting invalid block to the root chain
-    {:ok, _} = OMG.Eth.RootChain.submit_block(bad_block_hash, 2, 1)
-
     # checking if both machines and humans learn about the byzantine condition
     assert capture_log(fn ->
+             {:ok, %{"status" => "0x1", "blockNumber" => eth_height}} =
```

Comment:
This is unclear which of 2 function calls actually emit log event that we're expecting to capture?
Assuming from the change you're capturing to silence it from test output. Consider to add comment about which call we're checking log for and also we're silencing any other from appearing in the output.